Packs/CommonScripts/Scripts/SearchIncidentsV2/SearchIncidentsV2_test.py

```diff
@@ -41,3 +41,19 @@ def test_is_valid_args(_input, expected_output):
         output = False
 
     assert output == expected_output, f'is_valid_args({_input}) returns: {output}. expected: {expected_output}'
+
+
+def test_is_valid_args_with_int():
+    """
+    Given:
+        - an incident id from type int
+
+    When:
+        - running the script as a playbook task
+
+    Then:
+        - validating that the incident is is a valid input
+
+    """
+    output = is_valid_args({'id': 123})
+    assert output is True
```

Comment:
Isn't this more readable and Pythonic IYO?
```suggestion
    is_valid_id = is_valid_args({'id': 123})
    assert is_valid_id
```